com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -320,6 +320,9 @@ internal static void PerformDilation()
 
             if (dilationSettings.dilationDistance > 0.0f)
             {
+                // Make sure all pending unloads are done before we reset everything through ForceSHBands.
```

Comment:
@FrancescoC-unity can you validate this fix?
Basically some scenes are still in the pending unload list, we then reset the ref volume to a different size which makes subsequent unloading fail (out of bounds for flat array access).

Reply:
Uhm this is potentially dangerous I think, as we are flushing also the loading that are pending while we want to flush them only after we force the sh bands. 

We might need something that only flushes the unloading.

Reply:
Right, that makes sense. Shouldn't we be flushing everything at the start of baking anyway?
There's probably a better place for this I guess.

Reply:
Yes we can try to do that, we can try at the start of OnAdditionalProbesBakeCompleted , worth testing this a bit as I wonder if there are unwanted side effects we are not thinking about.